src/Components/VimList/Component_VimList.re

```diff
@@ -117,7 +118,8 @@ type msg =
 
 type outmsg =
   | Nothing
-  | Selected({index: int});
```

Comment:
I'm not 100% sure on the changing/removal of this name. `Clicked` and `DoubleClicked` don't have a vim equivalent, if I hit enter, I'm not clicking at all, so the name seems strange to me.

Reply:
I agree! I'd view the List as having events like 'Click', 'Double-Click', 'Enter' as input, and then `outmsg` as the produced effect.

Good point about the vim equivalent - I think to be consistent with NerdTree/Ex/etc, we'd want to have the keyboard behavior be to open a file.